yt/go/yt/integration/operations_test.go

```diff
@@ -131,6 +132,65 @@ func TestOperationWithStderr(t *testing.T) {
 	}
 }
 
+func TestSuspendOperation(t *testing.T) {
+	t.Parallel()
+
+	env := yttest.New(t)
+
+	ctx := ctxlog.WithFields(context.Background(), log.String("subtest_name", t.Name()))
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
+	defer cancel()
+
+	opSpec := map[string]any{
+		"tasks": map[string]any{
+			"main": map[string]any{
+				"job_count": 1,
+				"command":   "sleep 5m",
+			},
+		},
+	}
+	opID, err := env.YT.StartOperation(ctx, yt.OperationVanilla, opSpec, nil)
+	require.NoError(t, err)
+
+	// Wait for the job to start.
+	for idx := 0; idx < 10; idx++ {
+		res, err := env.YT.ListJobs(ctx, opID, &yt.ListJobsOptions{JobState: &yt.JobRunning})
+		require.NoError(t, err)
+		if len(res.Jobs) == 0 {
+			if idx == 9 {
+				t.Fatal("The job did not start")
+			}
+			time.Sleep(3 * time.Second)
+		} else {
+			break
+		}
+	}
+
+	// Suspend the operation.
+	err = env.YT.SuspendOperation(ctx, opID, &yt.SuspendOperationOptions{AbortRunningJobs: true})
+	require.NoError(t, err)
+
+	// Wait for the job to stop
+	for idx := 0; idx < 10; idx++ {
+		res, err := env.YT.ListJobs(ctx, opID, &yt.ListJobsOptions{JobState: &yt.JobRunning})
+		require.NoError(t, err)
+		if len(res.Jobs) != 0 {
+			if idx == 9 {
+				t.Fatal("The job did not stop")
+			}
+			time.Sleep(3 * time.Second)
+		} else {
+			break
+		}
+	}
+
+	// Check the operation status
+	opStatus, err := env.YT.GetOperation(ctx, opID, &yt.GetOperationOptions{})
+	require.NoError(t, err)
+	assert.Equal(t, yt.StateRunning, opStatus.State)
```

Comment:
There is a require.Equal, why can't we use it? Is a new dependency really needed?

Reply:
My bad, will fix

Reply:
btw, it would have not become a really new dependency in the project, unfortunately: https://github.com/search?q=repo%3Aytsaurus%2Fytsaurus+github.com%2Fstretchr%2Ftestify%2Fassert+path%3A%2F%5Eyt%5C%2Fgo%5C%2Fyt%5C%2Fintegration%5C%2F%2F&type=code